schemas/latest/dev-workspace-template-spec.json

```diff
@@ -6,7 +6,7 @@
     "attributes": {
       "description": "Map of implementation-dependant free-form YAML attributes.",
       "type": "object",
-      "additionalProperties": true
+      "additionalProperties": {}
```

Comment:
This looks like an error

Reply:
it's fixed in the first commit, see schemas/gen.go file. Sorry for squashing everything, otherwise it would be even difficult to review overall picture I believe.

So, AFAIU controller-gen changed their default so now they provide empty schema with XPreserveUnknownFields, which David (I don't know reason well, probably JSON schema does not have such field) ignores during generating JSON Schema.
but schema takes over allow in JSONSchemaPropsOrBool, so it led to empty object in additional properties.

Now, I ignore schema in additional properties if it does not define type, which is basic field.